api/replication.storage/v1alpha1/volumereplication_types.go

```diff
@@ -36,6 +36,25 @@ const (
 	ConditionValidated = "Validated"
 )
 
+// These are valid messages for various
+// conditions and states of Volume
+// replication.
+const (
+	MessagePromoted               = "volume is promoted"
```

Comment:
This does not explain more than Promoted=true. Maybe something like "volume promoted to primary and replicating to secondary volume"